ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/PropertyNamingRuleTest.kt

```diff
@@ -158,4 +159,50 @@ class PropertyNamingRuleTest {
             """.trimIndent()
         propertyNamingRuleAssertThat(code).hasNoLintViolations()
     }
+
+    @Nested
+    inner class `Given property is serialVersionUID` {
+        @Test
+        fun `Given property is present in companion object`() {
+            val code =
+                """
+                class Foo1 {
+                    companion object {
+                        private const val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo2 {
+                    companion object {
+                        private val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo3 {
+                    companion object {
+                        @JvmStatic private val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo4 {
+                    private val serialVersionUID: Long = 123
+                }
+                """.trimIndent()
+            propertyNamingRuleAssertThat(code).hasNoLintViolations()
+        }
+
+        @Test
+        fun `Given property defined object is private const`() {
+            val code =
+                """
+                object Foo1 {
+                    private const val serialVersionUID: Long = 123
+                }
+                object Foo2 {
+                    private val serialVersionUID: Long = 123
+                }
+                object Foo3 {
+                    @JvmStatic private val serialVersionUID: Long = 123
+                }
```

Comment:
What is the value of those additional test cases? You only want to verify that a const property with identifier serialVersionUID is not being reported as violation.

Reply:
Are you referring to `@JvmStatic`, I have added that to test slightly different PSI structure as well

Reply:
I am referring to both Foo2 and Foo3. Those cases were already not reported before because the const modifier was missing. Adding an extra exclusion based on name will never interfere with that.

Reply:
Updated